transactions/randomBeaconHistory/record_source_of_randomness.cdc

```diff
@@ -0,0 +1,19 @@
+import "RandomBeaconHistory"
+
+/// Commits the source of randomness for the requested block height from the RandomBeaconHistory Heartbeat
+///
+// TODO: Replace transaction parameter with randomSourceHistory() call
+transaction(sor: [UInt8]) {
```

Comment:
This transaction is not really needed, only the system transaction will be doing this.

Reply:
Got it, it's included here only for the purpose of testing until I can get tests running using the [system chunk transaction](https://github.com/onflow/flow-go/blob/1231763cb306d6d81cdf7c922c5761097d1dcfb9/fvm/blueprints/scripts/systemChunkTransactionTemplate.cdc#L19).